casanovo/casanovo.py

```
@@ -334,6 +356,133 @@ def train(
+            cfg.n_beams = n_beams
+
+            throughputs = []
+            peak_mem_mb = 0
+
+            total_iters = warmup + n_iter
+            for i in range(total_iters):
+                results_path = (
+                    output_path
+                    / f"{output_root_name}_bench_{batch_size}_{n_beams}_{i}.mztab"
+                )
+                if torch.cuda.is_available():
```

Ruff (0.15.10)

[error] Undefined name `torch`

(F821)